package.json

```diff
@@ -61,13 +53,10 @@
     "lint": "eslint --ext .js,.jsx src",
     "build": "npm run build:js && npm run build:scss",
     "build:js": "webpack -d",
-    "postbuild:js": "uglifyjs build/js/lyra.js -cm > build/js/lyra.min.js",
     "build:scss": "node-sass --include-path src/scss src/scss/main.scss build/css/main.css",
-    "postbuild:scss": "cleancss -o build/css/main.min.css build/css/main.css",
     "watch": "npm run watch:js & npm run watch:scss",
     "watch:js": "webpack -d --watch",
     "watch:scss": "nodemon -e src/scss -x \"npm run build:scss\"",
-    "test": "npm run test",
```

Comment:
Assuming @deathbearbrown will reinstate this in such a way that it actually does something as she gets Karma set up